db/migrate/20230711141927_create_users.rb

```diff
@@ -0,0 +1,12 @@
+class CreateUsers < ActiveRecord::Migration[7.0]
+  def change
+    create_table :users do |t|
+      t.string :name
+      t.string :photo
+      t.string :bio
+      t.integer :posts_count
```

Comment:
- [ ] **Optional:** It is good to stick with naming as used in the ERD because most of the time you'd work with large teams which use the ERD as a source of truth. Note this in future projects, except if the requirement was to personalize it.

![image](https://github.com/dev89s/blog-app/assets/27709832/57ca79c6-9fb1-4b9e-a158-41704f2a7b53)

ERD says `posts_counter`, yours says `posts_count`